deploy/helm/events-collector-config.yaml

```diff
@@ -6,7 +6,11 @@ exporters:
     headers:
       "Authorization": "Bearer ${SOLARWINDS_API_TOKEN}"
 extensions:
-  health_check: {}
+  health_check:
+    check_collector_pipeline:
+      enabled: true
```

Comment:
When I was playing with this health check, I was not able to configure it work correctly. It kept failing after each pass of the `interval` and the collector was being regularly restarted. In the otel-contrib repo, it says that the extension works only with some exporters. Does it actually work with the  `otlp` exporter?